pandas/io/parsers.py

```diff
@@ -944,17 +951,22 @@ def _clean_options(self, options, engine):
         sep = options["delimiter"]
         delim_whitespace = options["delim_whitespace"]
 
-        # C engine not supported yet
-        if engine == "c":
+        # arrow engine not supported yet
+        if engine == "arrow":
+            if self.chunksize is not None:
+                fallback_reason = f"the arrow engine does not support chunksize"
```

Comment:
Can we raise an error instead? I mean, if people explicitly specify `engine="pyarrow"`, they want that, and not the python engine (it's different for the C engine, since that is the default, and users typically don't specify this)

Reply:
And the same for the others below

Reply:
@jorisvandenbossche below that block of code is an if statement that automatically raises an exception.
` if fallback_reason and engine_specified:
            raise ValueError(fallback_reason)`

Reply:
Ah, OK! But then the line below `engine = "python"` is not really needed?